modules/ingester/instance.go

```diff
@@ -154,13 +172,49 @@ func newInstance(instanceID string, limiter *Limiter, overrides ingesterOverride
 	return i, nil
 }
 
-func (i *instance) PushBytesRequest(ctx context.Context, req *tempopb.PushBytesRequest) error {
+func (i *instance) PushBytesRequest(ctx context.Context, req *tempopb.PushBytesRequest) *tempopb.PushResponse {
+	var errors util.MultiError
+
+	maxLiveErrorTraces := make([]int32, 0)
```

Comment:
the compiler likely does this for us, but we should not call `make()` here so the slices are only allocated if an error occurs